charts/argo-templates/templates/_helpers.tpl

```diff
@@ -32,3 +32,19 @@ retryStrategy:
 {{ . }}
 {{- end }}
 {{- end }}
+
+{{/* Collect metrics about execution results, including success/failure counts. */}}
+{{- define "argo.collect-execution-metrics" }}
+- name: result-counter
```

Comment:
I think the Argo server might collect this by default, based on an example I saw in an issue. We should check what appears on `/metrics` before we add any new metrics to our existing workflows.